src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -326,6 +340,34 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
         return true;
     }
 
+    private ScheduleRunConfiguration getScheduleRunConfiguration(Boolean isRunUnmetered) {
+        ScheduleRunConfiguration configuration = new ScheduleRunConfiguration();
+        if (isRunUnmetered != null && isRunUnmetered) {
+            configuration.setBillingMethod(BillingMethod.UNMETERED);
+        } else {
+            configuration.setBillingMethod(BillingMethod.METERED);
+        }
+
+        // set a bunch of other default values as Device Farm expect these
+        configuration.setAuxiliaryApps(new ArrayList<String>());
```

Comment:
We should discuss this (outside of scope of issue). Setting all this garbage should **NOT** be required from the API/SDK if you just want to set one of these values.